README.rst

```diff
@@ -343,31 +343,11 @@ a request through the NI support portal at `ni.com <http://www.ni.com>`_.
 Bugs / Feature Requests
 =======================
 
-To report a bug or submit a feature request, please use the
+To report a bug or submit a feature request specific to Python, please use the
```

Comment:
I know it seems like a stretch... but let's avoid hearing about Python issues and say
> ...specific to NI Modular Instruments Python bindings (nimi-python), please use...

We should also add, if you agree, something along the lines of:
> For hardware support or any other questions not specific to this GitHub project, please visit [NI Community Forums](https://forums.ni.com/).